src/main/java/org/gatorgradle/config/GatorGradleConfig.java

```diff
@@ -216,6 +238,10 @@ public void parseHeader() {
       reportEndpoint = file.getHeader("lambda").asString();
     }
 
+    if (file.hasHeader("reflection")) {
+      reflection = readFile(file.getHeader("reflection").asString());
```

Comment:
As a rule, we shouldn't store the contents of files or other things that were retrieved in the config -- this can lead to some weird situations since the config is parsed before checks are run, and it's just a generally good convention. So I would suggest simply storing the actual value of the reflection field (the path to the file), and when you need to actually get the reflection in other tasks you should do the file reading there.